firmware/torpedoboard/code/app/messageHandler_componentSpecific.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * messageHandler_componentSpecific.c
+ *
+ *  Created on: Aug 12, 2020
+ *      Author: Poorna Chander Kalidas
+ */
+
+#include "messageHandler.h"
+#include <stdlib.h>
+#include <string.h>
+#include "debug.h"
+
+static void messageHandler_componentSpecific_messageReceivedCallback(const messageHandler_RXMessageChannel_E channel, const protocol_allMessages_U * const message);
+static void messageHandler_componentSpecific_messagePopulateCallback(const messageHandler_TXMessageChannel_E channel, protocol_allMessages_U * const message);
+
+extern const messageHandler_config_S messageHandler_config;
+const messageHandler_config_S messageHandler_config =
+{
+    .RXMessageConfig =
+    {
+
+        [MESSAGE_HANDLER_RX_CHANNEL_TESTER] =
+        {
+            .messageID = protocol_MID_MC_deviceName,
```

Comment:
This is the message ID for motor controller. Use the one for torpedo board. Create one if it doesn't exist.